mina/contracts/mina/src/sudoku.ts

```diff
@@ -26,7 +25,83 @@ class Sudoku extends Struct({
   }
 }
 
-class SudokuZkApp extends SmartContract {
```

Comment:
Probably we don't want to delete the old contract, and rather have an example of both options

Reply:
This `-` gets `+`'d back later in the file, so `SudokuZkApp` still exists. Its `submitSolution` signature does change, but I could also put it back and add the new one as `submitSolution2` or something.

Reply:
OK, both methods are now demonstrated.